src/constants/chainInfo.ts

```diff
@@ -148,7 +148,7 @@ const CHAIN_INFO: ChainInfoMap = {
   [SupportedChainId.POLYGON]: {
     networkType: NetworkType.L1,
     blockWaitMsBeforeWarning: ms`10m`,
-    bridge: 'https://wallet.polygon.technology/login',
```

Comment:
Thanks for catching this! We should probably link directly to the POS Bridge here instead of the zkEVM to avoid confusion until that's supported. Here is the direct link to the POS Bridge: https://wallet.polygon.technology/polygon/bridge

Reply:
For some reason that link still redirects me to the zkEVM bridge, however, I just updated the link in case that is just an issue on my end.

Reply:
yeah i'm also getting redirected here